app/main.py

```diff
@@ -99,7 +46,7 @@ def start_crawling(url: str = Form(...), num_links: int = Form(...), db: Session
                 create_screenshot(db, screenshot)
                 screenshot_names.append(screenshot_name)
             except Exception as e:
-                logging.error(f"Failed to create screenshot entry: {e}")
+                print(f"Failed to create screenshot entry: {e}")
```

Comment:
`print` is a bad choicee - use logging as before. If it's not showing in the console check in google why (hint: config logger)